index.js

```diff
@@ -4,23 +4,85 @@ var fs = require('fs');
 var backlightPath = '/sys/class/backlight/rpi_backlight';
 
 function writeValue(fileName, value) {
-    return new Promise(function (resolve, reject) {
-        var fullPath = path.join(backlightPath, fileName);
-        fs.writeFile(fullPath, value, function(err) {
-             if (err !== null) reject(err);
-             else resolve();
-        });
+  return new Promise((resolve, reject) => {
+    var fullPath = path.join(backlightPath, fileName);
+    fs.writeFile(fullPath, value, (err) => {
+      if (err !== null) reject(err);
+      else resolve();
```

Comment:
Looks like you've updated the indentation all through here. Can you leave it as 4 spaces please? Keeps the git history cleaner that way, and makes this diff easier to follow.

Reply:
Yep...sorry. I will pay more attention to the future to have a cleaner PR.

Reply:
Sure, no worries! I know this is lots of comments, but don't worry about it, it's a lot of code :smiley: